code/modules/reagents/chemistry/reagents/drink_base.dm

```diff
@@ -19,7 +19,7 @@
 	if(adj_drowsy)
 		M.AdjustDrowsy(adj_drowsy)
 	if(adj_sleepy)
-		update_flags |= M.AdjustSleeping(adj_sleepy, FALSE)
+		M.AdjustSleeping(adj_sleepy)
```

Comment:
You will need to go through all drinks that set `adj_dizzy`, `adj_drowsy`, `adj_sleepy` and make sure they're defined in seconds.